images/win/scripts/Installers/Install-AndroidSDK.ps1

```diff
@@ -0,0 +1,147 @@
+################################################################################
+##  File:  Install-AndroidSDK.ps1
+##  Desc:  Install and update Android SDK and tools
+################################################################################
+
+# install command-line tools
+$cmdlineToolsUrl = "https://dl.google.com/android/repository/commandlinetools-win-7302050_latest.zip"
+$cmdlineToolsArchPath = Start-DownloadWithRetry -Url $cmdlineToolsUrl -Name "cmdline-tools.zip"
+$sdkInstallRoot = "C:\Program Files (x86)\Android\android-sdk"
+$sdkRoot = "C:\Android\android-sdk"
+Expand-Archive -Path $cmdlineToolsArchPath -DestinationPath "${sdkInstallRoot}\cmdline-tools" -Force
+# cmdline tools should be installed in ${sdkInstallRoot}\cmdline-tools\latest\bin, but archive contains ${sdkInstallRoot}\cmdline-tools\bin 
+# we need to create the proper folder structure
+Rename-Item "${sdkInstallRoot}\cmdline-tools\cmdline-tools" "latest"
+# ANDROID_NDK_PATH/HOME should not contain spaces. Otherwise, the script ndk-build.cmd gives an error https://github.com/actions/virtual-environments/issues/1122
```

Comment:
could you please add a new line between comment and a previous command? the whole text looks more unreadable